src/main/java/me/totalfreedom/totalfreedommod/command/Command_namehistory.java

```diff
@@ -1,23 +1,23 @@
-package me.totalfreedom.totalfreedommod.command;
-
-import me.totalfreedom.totalfreedommod.rank.Rank;
-import me.totalfreedom.totalfreedommod.util.History;
-import org.bukkit.command.Command;
-import org.bukkit.command.CommandSender;
-import org.bukkit.entity.Player;
-
-@CommandPermissions(level = Rank.OP, source = SourceType.BOTH)
-@CommandParameters(description = "Check the name history of a specified player.", usage = "/<command> <username>", aliases = "nh")
-public class Command_namehistory extends FreedomCommand
-{
-    @Override
-    public boolean run(final CommandSender sender, final Player playerSender, Command cmd, String commandLabel, String[] args, boolean senderIsConsole)
-    {
-        if (args.length != 1)
-        {
-            return false;
-        }
-        History.reportHistory(sender, args[0]);
-        return true;
-    }
+package me.totalfreedom.totalfreedommod.command;
```

Comment:
This entire file hasn't changed, and doing this is gonna break the Git history (OR make it difficult to work with)